src/components/Sidebar/components/CloseButton/CloseButton.js

```diff
@@ -52,18 +51,18 @@ const visibleStyles = ({ visible }) =>
     opacity: 1;
   `;
 
-const AccessibilityLabel = styled.span`
-  ${hideVisually()};
-`;
-
 const FloatingButton = styled.button`
   ${baseStyles};
   ${visibleStyles};
 `;
 
 const CloseButton = ({ visible, closeButtonLabel, onClick, ...props }) => (
-  <FloatingButton visible={visible} onClick={onClick} {...props}>
-    <AccessibilityLabel>{closeButtonLabel}</AccessibilityLabel>
+  <FloatingButton
+    visible={visible}
+    onClick={onClick}
+    {...props}
+    aria-label={closeButtonLabel}
```

Comment:
Why did you switch to `aria-label` here? It has worse screen reader support than an explicit, visually-hidden label.

Reply:
The reason is that accessibility check on `CloseButton.spec.js` after axe update started to fail.

Reply:
Hm, could you please try switching to the `IconButton` component instead? It's accessible out of the box.

Reply:
👍 will check this out

Reply:
resolved